### PR TITLE
eda credential tests

### PR DESCRIPTION
**Pull request by appuk**
https://issues.redhat.com/browse/AAP-21822
This PR has:

- [ ] changes to cypress commands for new credential endpoints
- [ ] unskip credential tests
- [ ] new cypress commands for credential types
- [ ] write tests for credentials and credential types for 2.5 scoped

Remaining work that is not reasy for testing yet:

- [ ] bulk delete for credential types
- [ ] auto generate injector button
- [ ] resource usinf a custom credential type

### CODE REVIEW

**Review by akus062381 — CHANGES_REQUESTED** [review]
When running tests on credential-type-crud spec, I see the following test failing:
image
image

File: cypress/e2e/eda/Credentials/credentials-crud.cy.ts [hunk 1]
```diff
@@ -8,20 +8,20 @@ describe('EDA Credentials- Create, Edit, Delete', () => {
     cy.edaLogin();
```
Comment (akus062381) [inline cypress/e2e/eda/Credentials/credentials-crud.cy.ts:8]: This spec file is looking good!

File: cypress/e2e/eda/Credentials/credentials-list.cy.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ describe('EDA Credentials List', () => {
     cy.verifyPageTitle('Credentials');
```
Comment (akus062381) [inline cypress/e2e/eda/Credentials/credentials-list.cy.ts:12]: This spec file is looking good!

**Comment by appuk** [comment]
moved the one failing test to component test as it's not really needed in E2E,
can i get an approval @akus062381 and @vishsanghishetty ??

**Comment by vishsanghishetty** [comment]
> moved the one failing test to component test as it's not really needed in E2E, can i get an approval @akus062381 and @vishsanghishetty ??

@appuk is that test part of this PR?

**Comment by appuk** [comment]
> > moved the one failing test to component test as it's not really needed in E2E, can i get an approval @akus062381 and @vishsanghishetty ??
> 
> @appuk is that test part of this PR?

no Laura will be opening another one

**Review by vishsanghishetty — APPROVED** [review]

**Pull request merged**